tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerDelegate.java

```diff
@@ -251,4 +264,70 @@ public void setUsername( String username )
         setAttribute( ATTR_USERNAME, username );
     }
 
-}
\ No newline at end of file
+    public void setHttpPort( String httpPort )
+    {
+        setAttribute( ATTR_HTTP_PORT, httpPort );
+
+        PortalRuntime runtime =
+            (PortalRuntime) getServer().getRuntime().loadAdapter( PortalRuntime.class, new NullProgressMonitor() );
+
+        if( runtime != null )
```

Comment:
This code only makes sense for Tomcat.  So this should be moved to a tomcat specific class.